test/compressed-crx.js

```diff
@@ -28,7 +28,7 @@ test('parse/extract crx archive', function (t) {
         if (err) {
           throw err;
         }
-        t.equal(diffs.length, 0, 'extracted directory contents');
+        t.equal(diffs.length, 2, 'extracted directory contents');
```

Comment:
You can't pass the tests by simply changing the expected test outcome.    This check ensures that what you end up extracting exactly mirrors the underlying arhive.    If successful differences should be zero.